app/views/layouts/_navigation_links.html.erb

```diff
@@ -1,23 +1,39 @@
-<% if session[:line] %>
-  <ul class="nav navbar-nav navbar-left">
-    <li class="navbar-text"><%= current_line_display %></li>
-    <% if ENV['CM_RESOURCES_URL'] %>
```

Comment:
Can we keep the resources URL here, rather than moving it into the dropdowns? Want to make sure it's super front and center for CMs to hit.